taxi/tests/tests_models.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+# Create your tests here.
+class ModelsTestCase(TestCase):
+    def test_manufacturer_str_and_dani(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test",
+            country="Ukraine",
+        )
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )
+        self.assertEqual(manufacturer.name, "test")
+        self.assertEqual(manufacturer.country, "Ukraine")
+
+    def test_car_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test",
+            country="Ukraine",
+        )
+        car = Car.objects.create(
+            model="test",
+            manufacturer=manufacturer,
+        )
+        self.assertEqual(str(car), "test")
+
+    def test_driver_str_and_dani(self):
```

Comment:
There is a typo in the method name `test_driver_str_and_dani`. It should be `test_driver_str` to correctly reflect the purpose of the test.